youtube_transcript_api/_transcripts.py

```diff
@@ -314,16 +334,28 @@ def translate(self, language_code):
             language_code,
             True,
             [],
+            preserve_formatting=self.preserve_formatting,
         )
 
 
 class _TranscriptParser(object):
-    HTML_TAG_REGEX = re.compile(r'<[^>]*>', re.IGNORECASE)
+    def __init__(self, preserve_formatting=False):
+        self.preserve_formatting = preserve_formatting
```

Comment:
We do not need to keep this state around, since it is not needed after the object has been initialized